wcfsetup/install/files/lib/action/DeleteSessionAction.class.php

```diff
@@ -0,0 +1,70 @@
+<?php
+namespace wcf\action;
+use wcf\system\exception\IllegalLinkException;
+use wcf\system\session\SessionHandler;
+use wcf\system\WCF;
+use wcf\util\JSON;
+use wcf\util\StringUtil;
+
+/**
+ * Deletes a specific user session.
+ *
+ * @author	Joshua Ruesweg
+ * @copyright	2001-2020 WoltLab GmbH
+ * @license	GNU Lesser General Public License <http://opensource.org/licenses/lgpl-license.php>
+ * @package	WoltLabSuite\Core\Action
+ */
+class DeleteSessionAction extends AbstractAction {
+	/**
+	 * @inheritDoc
+	 */
+	public $loginRequired = true;
+	
+	/**
+	 * @var string
+	 */
+	public $sessionID;
```

Comment:
```suggestion
	private $sessionID;
```
I don't want plugins touching that value.